packages/nimble-components/src/wafer-map/index.ts

```diff
@@ -138,7 +147,9 @@ export class WaferMap extends FoundationElement {
     @observable public hoverDie: WaferMapDie | undefined;
 
     @observable public highlightedTags: string[] = [];
-    @observable public dies: WaferMapDie[] = [];
+    @observable public dies: WaferMapDie[] | undefined;
```

Comment:
We can leave this initialized to empty array now right? the condition is that diesTable is assigned to switch the renderer. Then we can avoid almost all the null checks being added.

```suggestion
    @observable public dies: WaferMapDie[] = [];
```